uat/testing-features/src/main/java/com/aws/greengrass/LocalDeploymentSteps.java

```diff
@@ -162,24 +193,27 @@ private void installComponent(List<String> component, List<Map<String, String>>
         }
     }
 
-    private String getCliUpdateConfigArgs(String componentName, List<Map<String, String>> configuration) throws IOException {
+    private void updateComponent(String component, List<Map<String, String>> configuration)
+            throws InterruptedException, IOException {
+            String componentName = component;
```

Comment:
What value is this adding? You could replace the parameter name and avoid this reassignment

Reply:
done